x/bounty/client/rest/query.go

```diff
@@ -1,22 +1,115 @@
 package rest
```

Comment:
this is a legacy route so it probably shouldn't be used/supported, but let's keep it since we already have them

Reply:
Yes, currently only the rest query related to programs is implemented

Reply:
I think we can remove LegacyQuerier if not needed. Branches can be kept and can be quickly added